### PR TITLE
Fix typo, pythonic equality, fix import for pyemd.

### PR DESCRIPTION
**Pull request by kishan3**
Fix for issue #2228

### CODE REVIEW

File: gensim/models/keyedvectors.py [hunk 1]
```diff
@@ -765,9 +764,9 @@ def wmdistance(self, document1, document2):
         if diff1 > 0 or diff2 > 0:
             logger.info('Removed %d and %d OOV words from document 1 and 2 (respectively).', diff1, diff2)
 
-        if len(document1) == 0 or len(document2) == 0:
+        if not len(document1) or not len(document2):
             logger.info(
-                "At least one of the documents had no words that werein the vocabulary. "
+                "At least one of the documents had no words that wherein the vocabulary. "
```
Comment (piskvorky) [inline gensim/models/keyedvectors.py]: That sentence doesn't make sense. I think the intent here was `were in`.

File: gensim/models/keyedvectors.py [hunk 2]
```diff
@@ -765,9 +764,9 @@ def wmdistance(self, document1, document2):
         if diff1 > 0 or diff2 > 0:
             logger.info('Removed %d and %d OOV words from document 1 and 2 (respectively).', diff1, diff2)
 
-        if len(document1) == 0 or len(document2) == 0:
+        if not len(document1) or not len(document2):
```
Comment (piskvorky) [inline gensim/models/keyedvectors.py]: `not x:` is how you test for emptiness in Python.

File: gensim/models/keyedvectors.py [hunk 3]
```diff
@@ -752,8 +752,7 @@ def wmdistance(self, document1, document2):
             If `pyemd <https://pypi.org/project/pyemd/>`_  isn't installed.
 
         """
-        if not PYEMD_EXT:
-            raise ImportError("Please install pyemd Python package to compute WMD.")
+        from pyemd import emd
```
Comment (piskvorky) [inline gensim/models/keyedvectors.py]: How does this affect existing code that relied on `PYEMD_EXT`? Do we still even need `PYEMD_EXT`, where is it used?

File: gensim/models/keyedvectors.py (hunk 2 above)
Comment (kishan3) [reply gensim/models/keyedvectors.py]: Ok, so are intending to use `is` keyword here? like `len(document1) is 0` ?

File: gensim/models/keyedvectors.py (hunk 3 above)
Comment (kishan3) [reply gensim/models/keyedvectors.py]: we can remote `PYEMD_EXT` from wherever it is used in the project with proper imports. Is that what you mean here?

Comment (menshikh-iv) [reply gensim/models/keyedvectors.py]: Yes. As I remember, this also used in tests, but this isn't imported from here, i.e. filled directly in test (I don't think that we need the same change in the tests, only here).

File: gensim/models/keyedvectors.py (hunk 2 above)
Comment (menshikh-iv) [reply gensim/models/keyedvectors.py]: not `is`, by `x` @piskvorky mean right side, like `not document1`, etc.

Comment (kishan3) [reply gensim/models/keyedvectors.py]: Ok got you! Thanks @menshikh-iv

File: gensim/models/keyedvectors.py (hunk 3 above)
Comment (kishan3) [reply gensim/models/keyedvectors.py]: Ok sir, so please let me know what else to do. I have removed the import from the start now.

**Pull request closed without merging**